mailpoet/assets/js/src/automation/editor/store/reducer.ts

```diff
@@ -50,6 +50,23 @@ export function reducer(state: State, action: Action): State {
           [action.stepType.key]: action.stepType,
         },
       };
+    case 'UPDATE_STEP_NAME': {
+      const step = {
+        ...state.workflowData.steps[action.stepId],
+        name: action.name,
+      };
+      return {
+        ...state,
+        workflowData: {
+          ...state.workflowData,
+          steps: {
+            ...state.workflowData.steps,
+            [action.stepId]: step,
+          },
+        },
+        selectedStep: step,
+      };
+    }
```

Comment:
This looks almost identical to `UPDATE_STEP_ARGS`. Is there a reason we can't simply use `UPDATE_STEP_ARGS` instead? And is there a reason this action doesn't mark the workflow as being unsaved like `UPDATE_STEP_ARGS` does on L87?
```
workflowSaved: false
```